client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -101,14 +101,19 @@ function NonBreakingSpace() {
   return <>{"\u00A0"}</>;
 }
 
-function labelFromString(version: string | boolean | null | undefined) {
+function labelFromString(
+  version: string | boolean | null | undefined,
+  removed?: boolean
+) {
   if (typeof version !== "string") {
     return <>{"?"}</>;
   }
   if (!version.startsWith("≤")) {
     return <>{version}</>;
   }
-  const title = `Supported in version ${version.slice(1)} or earlier.`;
+  const actionDescription = removed ? "ended" : "started";
+  const versionDescription = `version ${version.slice(1)} or earlier`;
```

Comment:
Assuming `version` has a value of `≤ 79` there would be a doubled space here.
Is it guaranteed to be in shape of `≤79`?
(79 being a random number)